src/cli/bootstrap/keys.ts

```diff
@@ -45,3 +45,12 @@ export async function updateAccountKey(account: NameType, newPublicKey: PublicKe
     await eosioContract.updateauth(account.toString(), 'active', 'owner', authority, signer);
     await eosioContract.updateauth(account.toString(), 'owner', 'owner', authority, signer);
 }
+
+export async function updateControllByAccount(account: NameType, owner: string, addCodePermission = false) {
+    const authority = Authority.fromAccount({ actor: owner.toString(), permission: 'owner' });
+
+    if (addCodePermission) authority.addCodePermission(account.toString());
+
+    await eosioContract.updateauth(account.toString(), 'active', 'owner', authority, signer);
+    await eosioContract.updateauth(account.toString(), 'owner', 'owner', authority, signer);
+}
```

Comment:
```ts
    const ownerAuthority = Authority.fromAccount({ actor: owner.toString(), permission: 'owner' });
    const activeAuthority = Authority.fromAccount({ actor: owner.toString(), permission: 'active' });

    if (addCodePermission) ownerAuthority.addCodePermission(account.toString());
    if (addCodePermission) activeAuthority.addCodePermission(account.toString());

    await eosioContract.updateauth(account.toString(), 'active', 'owner', activeAuthority, signer);
    await eosioContract.updateauth(account.toString(), 'owner', 'owner', ownerAuthority, signer);
```

Reply:
We want the owner permission to be controlled by the controlling accounts owner permission
and the active mission to be controlled by the controlling  accounts active permission